core/src/test/scala/com/salesforce/op/ModelInsightsTest.scala

```diff
@@ -761,4 +774,29 @@ class ModelInsightsTest extends FlatSpec with PassengerSparkFixtureTest with Dou
     absError / bigCoeffSum < tol shouldBe true
     absError2 / smallCoeffSum < tol shouldBe true
   }
+
+  val tol2 = 0.1
```

Comment:
Can we put all of these inside the test(s) that use them or up with the other fields?